modules/cnf-image-based-upgrade-prep-resources.adoc

```diff
@@ -0,0 +1,431 @@
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-app-lvms
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3"
+spec:
+  backupName:
+    backup-example-app-lvms
+  restorePVs: true
+  restoreStatus:
+    includedResources:
+      - logicalvolumes
+----
+
+. Define the apply order for the OADP Operator in the `Restore` CRs by using the `lca.openshift.io/apply-wave` field:
++
+--
+.Example OADP CRs without logical volumes
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: backup-example-app
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  excludedClusterScopedResources:
+  - persistentVolumes
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-acm-klusterlet
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "1"
+spec:
+  backupName:
+    backup-acm-klusterlet
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-app
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "2"
+spec:
+  backupName:
+    backup-example-app
+----
+
+.Example OADP CRs with logical volumes
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: backup-example-app-lvms
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  includedClusterScopedResources:
+  - persistentVolumes
+  - volumesnapshotcontents
+  - logicalvolumes.topolvm.io
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-lvmcluster
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "2" <1>
+spec:
+  backupName:
+    backup-example-lvmcluster
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-app-lvms
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3" <3>
+spec:
+  backupName:
+    backup-example-app-lvms
+  restorePVs: true
+  restoreStatus:
+    includedResources:
+      - logicalvolumes
+----
+<1> Ensure that the `lca.openshift.io/apply-wave` annotation is numerically lower in your {lvms} `Backup` and `Restore` CRs than that of the application so that they are restored before your applications.
+
+[NOTE]
+====
+If you do not define the `lca.openshift.io/apply-wave` annotation in the `Backup` or `Restore` CRs, they will be applied together.
```

Comment:
```suggestion
If you do not define the `lca.openshift.io/apply-wave` annotation in the `Restore` CRs, they will be applied together.
```
the `lca.openshift.io/apply-wave` annotation is only applicable in Restore CRs, not needed by BAckup CRs